provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs

```diff
@@ -78,14 +78,7 @@ public ProvisioningTransportHandlerHttp()
 
                 Logging.Associate(authStrategy, this);
 
-                var httpClientHandler = new HttpClientHandler
-                {
-                    // Cannot specify a specific protocol here, as desired due to an error:
```

Comment:
These are all notes I left for the Jedi work I did this year, that we're going to have to finish this next year. I would prefer you leave them here.